applications/luci-app-https-dns-proxy/htdocs/luci-static/resources/view/https-dns-proxy/overview.js

```diff
@@ -81,7 +81,7 @@ return view.extend({
 			"dnsmasq_config_update_option",
 			_("Update DNSMASQ Config on Start/Stop"),
 			_(
-				"If update option is selected, the %s'DNS Forwards' section of DHCP and DNS%s will be automatically updated to use selected DoH providers (%smore information%s)."
+				"If update option is selected, the %s'DNS forwardings' section of DHCP and DNS%s will be automatically updated to use selected DoH providers (%smore information%s)."
```

Comment:
I pushed out the change to master recently for 'DNS Forwards' and this change also seemed like it should match. Does this not refer to the same main DHCP section under an interface?

Reply:
Wait, did you change it from `DNS forwardings` to `DNS Forwards` in master?

Reply:
Yes. 1b18cea61d99ca75581dc09010b118a86ac8dc28 and 
22467ec732cca0f4f259597535bd304bb5983075

Reply:
Oh, is there a plan to cherry-pick the luci-base update for 23.05 and 22.03? It would be hellish to maintain different strings for different branches for me?

Reply:
I'll get those in right away.

Reply:
Thank you so much for bringing it up, I've updated this PR and I'll redo release cherrypicks as well!

Reply:
OK, all ready. Just git fetch for master and branches.